projects/coordinator/src/main/java/org/batfish/coordinator/WorkMgrService.java

```diff
@@ -555,6 +555,46 @@ public JSONArray getAnswer(
       }
    }
 
+   @POST
+   @Path(CoordConsts.SVC_RSC_GET_CONTAINER)
+   @Produces(MediaType.APPLICATION_JSON)
+   public JSONArray getContainer(
```

Comment:
getting back to your response, @kidsbear, i see the difference between the two functions. yes, being able to check if a containerName exists without having access to it is useful (and not supported by listContainers).

and by that token, the purpose of this function should stay as something close to doesContainerExist() as no other info should be provided with clients with API keys that don't have access to the container.